### DISCUSSION TITLE
Version 0.7

### DISCUSSION THREAD

**Discussion by lpasselin** [facebookresearch/detectron2#4316] [Q&A]
Are there any plan to publish compiled binaries for a new version soon?
We hope to see this new release with the black version fix https://github.com/facebookresearch/detectron2/pull/4293

Let me know if I can help.
Thanks!

**Comment by ashnair1** [comment]
@ppwwyyxx Is there a plan for 0.7 release in the short term?

**Reply by ppwwyyxx** [reply]
I no longer work on this project, but I guess not given no release in the past year.

**Reply by yiftachbeer** [reply]
Any news on this? New code is still added to the repo, would be useful to have an official version.

**Comment by johnnynunez** [comment]
anyone accept my PR:
https://github.com/facebookresearch/detectron2/pull/4740 @ppwwyyxx @wat3rBro @thiagocrepaldi 
It would incredible if we have the last binaries only with pip install detectron2.
Most universities use detectron2 to teach their students.
